src/renovate.ts

```diff
@@ -35,6 +39,18 @@ class Renovate {
     }
   }
 
+  /**
+   * Fetch the host docker group on github.
+   * Container user needs this to access the docker socket.
+   */
+  private async getDockerGroup(): Promise<string> {
```

Comment:
I don't see why this should be an `async` function. As we're awaiting reading the file I think we might as wel just use `fs.readFileSync`. I suggest the following. I can't find how I can make this as suggesting in GitHub but if you agree I can push this as change.

```diff
@@ -20,7 +20,7 @@ class Renovate {
   }
 
   async runDockerContainer(): Promise<void> {
-    const group = await this.getDockerGroup();
+    const group = this.getDockerGroup();
     const commandArguments = [
       '--rm',
       `--env ${this.configFileEnv}=${this.configFileMountPath()}`,
@@ -43,8 +43,8 @@ class Renovate {
    * Fetch the host docker group on github.
    * Container user needs this to access the docker socket.
    */
-  private async getDockerGroup(): Promise<string> {
-    const groups = await fs.promises.readFile('/etc/group', {
+  private async getDockerGroup(): string {
+    const groups = fs.readFileSync('/etc/group', {
       encoding: 'utf-8',
     });
     const [, group] = /^docker:x:([1-9][0-9]*):$/m.exec(groups);
```

Reply:
I simply like the always async paradigm of node.